templates/cover-template.html

```diff
@@ -0,0 +1,16 @@
+<script type="x-tmpl-mustache">
+<div class="content-container">
+<div class="content-wrapper">
+<p>{{ metadata.presenterName }}</p>
+{{ content }}
+</div>
+</div>
+<footer>
+<div class="footer-content">
+<p>{{ metadata.styles }}</p>
+</div>
+<div class="footer-logo">
+<img data-src="{{ metadata.logo }}" alt="Logo">
+</div>
+</footer>
+</script>
```

Comment:
indentation should not be a problem in a template, right?
```suggestion
<script type="x-tmpl-mustache">
  <div class="content-container">
    <div class="content-wrapper">
      <p>{{ metadata.presenterName }}</p>
      {{ content }}
    </div>
  </div>
  <footer>
    <div class="footer-content">
      <p>{{ metadata.styles }}</p>
    </div>
    <div class="footer-logo">
      <img data-src="{{ metadata.logo }}" alt="Logo">
    </div>
  </footer>
</script>
```

Reply:
indentation would not work for the template.